mw-webapp/src/logic/wayPage/jobTags/JobTags.module.scss

```diff
@@ -1,6 +1,6 @@
 @import "src/styles/variables";
 
-$jobTagsBlockWidth: 130px;
+$jobTagsBlockWidth: 150px;
```

Comment:
why not the same as for statistics or metrics?

Reply:
Because for tags 30% os screen is too much